src/main/java/control/CreationModalViewController.java

```diff
@@ -214,21 +211,6 @@ private void createDirectorShot(MouseEvent event) {
         // TODO: Implement adding a DirectorShot
```

Comment:
Please don't remove these temporary variables. These are placeholders for when the director shot creation is implemented